probatus/feature_elimination/feature_elimination.py

```diff
@@ -160,7 +170,7 @@ def _check_if_fitted(self):
 
 
     @staticmethod
-    def _preprocess_data(X):
+    def _preprocess_data(X, verbose=0):
```

Comment:
this is a private method. Why not remove the @staticmethod and use `self.verbose`?

Reply:
This function will be moved to utils in #7 . For now i will merge it like this.